eZ/Publish/Core/MVC/Symfony/Templating/Tests/Twig/Extension/ContentExtensionTest.php

```diff
@@ -48,7 +48,8 @@ public function getExtensions()
                     array(),
                     $this->getMock('Psr\Log\LoggerInterface')
                 ),
-                $this->fieldHelperMock
+                $this->fieldHelperMock, 
```

Comment:
@iosolinf: you just have a trailing whitespace at the end of this line :) You can use [PHP Coding Standards Fixer](http://cs.sensiolabs.org/) v1.11.x with config file `.php_cs` provided in this repository to fix any CS issues in php files.